xsnippet/api/default.conf

```diff
@@ -15,6 +15,22 @@ host = 127.0.0.1
 port = 8000
 
 
+; ACCESS LOG FORMAT
```

Comment:
Oh gosh, we've got to move away from conf and use env variables as soon as possible, or it will be too late. :)